src/api/kube.ts

```diff
@@ -1875,30 +1889,40 @@ export class KubeHelper {
     }
   }
 
-  async createCheClusterIssuer(cheClusterIssuerYamlPath: string): Promise<void> {
+  async listClusterIssuers(version: string, labelSelector?: string): Promise<any[]> {
     const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
 
-    const cheClusterIssuer = this.safeLoadFromYamlFile(cheClusterIssuerYamlPath)
+    let res
     try {
-      await customObjectsApi.createClusterCustomObject('cert-manager.io', 'v1alpha2', 'clusterissuers', cheClusterIssuer)
+      res = await customObjectsApi.listClusterCustomObject('cert-manager.io', version, 'clusterissuers', undefined, undefined, undefined, labelSelector)
     } catch (e) {
       throw this.wrapK8sClientError(e)
     }
+
+    if (!res || !res.body) {
+      throw new Error('Unable to get cluster issuers list')
+    }
+    const clusterIssuersList: {items?: any[]} = res.body
+
+    return clusterIssuersList.items || []
```

Comment:
```suggestion
    return res.body.items || []
```

Reply:
We imply `items` field at by `const clusterIssuersList: {items?: any[]} = res.body`. I think it should be kept.